doc/man3/SSL_set_bio.pod

```diff
@@ -33,55 +33,62 @@ functions.
 SSL_set_bio() does a similar job as SSL_set0_rbio() and SSL_set0_wbio() except
 that it connects both the B<rbio> and the B<wbio> at the same time. This
 function transfers the ownership of B<rbio> and B<wbio> to B<ssl> except that
-the rules for this are much more complex. For this reason this function is
-considered a legacy function and SSL_set0_rbio() and SSL_set0_wbio() should be
-used in preference. The ownership rules are as follows:
+the rules for this are much more complex and are defined as follows:
+The ownership rules are as follows:
```

Comment:
Oh - I see you added a note at the end. I think that note should be at the beginning somewhere.